egs2/fsc_unseen/asr1/README.md

```diff
@@ -0,0 +1,35 @@
+<!-- Generated by ./scripts/utils/show_asr_result.sh -->
+# RESULTS
+
+## Environments
+- date: `Mon Oct 11 13:11:36 2021 -0400`
+- python version: `3.9.5 (default, Jun  4 2021, 12:28:51) [GCC 7.5.0]`
+- espnet version: `espnet 0.10.3a2`
+- pytorch version: `pytorch 1.8.1+cu102`
+- Git hash: `4e7d2ba3510463ae744d1a6d98f18388ad929a9d`
+  - Commit date: `Mon Nov 8 16:28:44 2021 -0500`
+- Pretrained Model
+  - Zenodo: https://zenodo.org/record/5655832
```

Comment:
How about adding a model to HF hub as well and add an HF hub link?

Reply:
Will do!